resources/files/ruby_219/rbconfig/rbconfig-aarch64-redhat-linux.rb

```diff
@@ -0,0 +1,268 @@
+
```

Comment:
As we're adding things under `resources/files/ruby_*` and `resources/patches/*`, should we also remove the corresponding "source"  files from puppet-agent?